src/app/features/courses/containers/courses-page/courses-page.component.ts

```diff
@@ -1,8 +1,94 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { CoursesItemModel } from '../../models/courses-item.model';
 
 @Component({
   selector: 'app-courses-page',
   templateUrl: './courses-page.component.html',
   styleUrls: ['./courses-page.component.scss']
 })
-export class CoursesPageComponent {}
+
+export class CoursesPageComponent implements OnInit {
+  coursesList: CoursesItemModel[];
+  clickedTimes = 0;
+
+  ngOnInit() {
+    this.coursesList = [
```

Comment:
minor: Create separate file for mock data, and then just import it into this file